nf_core/pipeline-template/conf/base.config

```
@@ -15,7 +15,7 @@ process {
     memory = { check_max( 6.GB * task.attempt, 'memory' ) }
     time   = { check_max( 4.h  * task.attempt, 'time'   ) }
 
-    errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+    errorStrategy = { task.exitStatus in 130..145 ? 'retry' : 'finish' }
```

```suggestion
    def acceptedErrors = 130..145
    errorStrategy = { task.exitStatus in acceptedErrors+104 ? 'retry' : 'finish' }
```
This should work, I think the only way to add a value to a range is by saving it to a variable.

---

Thanks! I just had a quick play in `nextflow console` and I think just using brackets works too:

```suggestion
    errorStrategy = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
```

Haven't tested this code in context ☝🏻  but this works:

```groovy
println(((130..145)+104))
println(129 in ((130..145)+104) ? "yes" : "no")
```
```console
[130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 104]
no
```